src/main/java/by/onliner/cucumber/steps/LogInSteps.java

```diff
@@ -0,0 +1,42 @@
+package by.onliner.cucumber.steps;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+public class LogInSteps extends AbstractSteps{
+
+    @Given("Open log-In page")
+    public void openLogInPage() {
+        mainPage.getMenu().openLogInPage();
+    }
+
+
+    @When("Click log-In form button")
+    public void clickLogInFormButton() {
+        logInPage.clickLogInFormButton();
+    }
+
+    @Then("The actual field warning text should much expected warning text {string}")
+    public void theActualLogInWarningTextShouldMuchExpectedWarningText(String expectedMessage) {
```

Comment:
Update method's name

Reply:
Renamed to isTheActualLogInWarningTextMuchTheExpectedWarningText